internal/driver/testdata/testflame.js

```diff
@@ -0,0 +1,86 @@
+(function () {
+  const test = new TestFixture();
+  const chart = document.getElementById("stack-chart");
+  if (!chart) {
+    test.err("could not find stack-chart");
+    return;
+  }
+  const chartRect = chart.getBoundingClientRect();
+
+  // Create map from box text to DOM element.
+  // TODO: Generalize to support multiple boxes for a given piece of text.
+  let boxMap;
+  fetchBoxes();
+  function fetchBoxes() {
+    boxMap = new Map();
+    const boxes = document.querySelectorAll(".boxbg");
+    for (let box of boxes) {
+      const text = box.innerText;
+      boxMap.set(text, box);
+    }
+  }
+
+  // rect gets the bounding box for box with text t.
+  function rect(t) {
+    const elem = boxMap.get(t);
+    if (!elem) test.err("did not find", t);
```

Comment:
Should we return here? Otherwise it seems we'd be dereferencing a null in the call in the next line.

Reply:
Done.